src/common/storage/src/stage.rs

```diff
@@ -285,7 +285,7 @@ fn blocking_list_files_with_pattern(
 
     // path is a dir
     let mut files = Vec::new();
-    let list = operator.list(path)?;
+    let list = operator.scan(path)?;
```

Comment:
BTW, What's the difference between `list` and `scan`?

Reply:
list in fs is `list_dir` not recursive